src/elements/content-uploader/progressCellRenderer.js

```diff
@@ -5,18 +5,22 @@
 
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
+import Browser from '../../utils/Browser';
 import messages from '../common/messages';
 import ItemProgress from './ItemProgress';
+
 import {
     ERROR_CODE_UPLOAD_FILE_SIZE_LIMIT_EXCEEDED,
     ERROR_CODE_ITEM_NAME_IN_USE,
     ERROR_CODE_ITEM_NAME_INVALID,
     ERROR_CODE_UPLOAD_PENDING_APP_FOLDER_SIZE_LIMIT,
     ERROR_CODE_UPLOAD_STORAGE_LIMIT_EXCEEDED,
     ERROR_CODE_UPLOAD_CHILD_FOLDER_FAILED,
-    STATUS_ERROR,
+    ERROR_CODE_UPLOAD_BAD_DIGEST,
+    ERROR_CODE_UPLOAD_FAILED_PACKAGE,
```

Comment:
Minor: can we alphabetize these imports?

Reply:
Same as above; you will note that some are already out of sequence, unfortunately. we can have a "best practices" PR to clean them up so future changes conform to the tidy content.